test/FunctionsV2/WebJobs.Extensions.DurableTask.Tests.V2.csproj

```diff
@@ -16,11 +16,11 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="4.19.4" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.37" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.39" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.1.3" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" Version="3.0.33" />
     <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="2.0.65" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
```

Comment:
why did we make these updates? Shouldn't matter, since it's a test project, but still curious

Reply:
There is a downgrade error at my local end for this test file after I upgrade the framework. So I did this change..

Reply:
This is expected: this project has a p2p reference on our WebJobs, which now transitively brings in Loggin/6.0.0. You could alternatively just remove this explicit reference and let the transitive reference control the version.